src/fake_bpy_module/transformer/data_type_refiner.py

```diff
@@ -471,8 +543,10 @@ def _get_data_type_options(
                 for opt in options:
                     if opt not in ("optional", "readonly", "never none"):
                         has_unknown_option = True
-                        output_log(LOG_LEVEL_WARN,
-                                   f"Unknown option '{opt}' is found from {dtype_str}")
+                        # output_log(
+                        #     LOG_LEVEL_WARN,
+                        #     f"Unknown option '{opt}' is found from {dtype_str}",
+                        # )
```

Comment:
Why are these lines commented out?

Reply:
I commented these lines for a test. Will revert.